chi/hardware.py

```diff
@@ -98,6 +100,42 @@ def get_host_id(items, target_uid):
             blazarclient.host.get_allocation(host_id), minimum_hours
         )
 
+    def _ipython_display_(self):
+        """
+        Displays information about the node. This function is called passively by the Jupyter display system.
+        """
+
+        layout=Layout(padding="4px 10px")
+        style = {
+            'description_width': 'initial',
+            'background': '#d3d3d3',
+            'white_space': 'nowrap'
+        }
+
+        reservable_style = {
+            'description_width': 'initial',
+            'background': ' #a2d9fe',
+            'white_space': 'nowrap'
+        }
+        
+        if not self.reservable:
+            reservable_style['background'] = '#f69084'
+    
+        box = Box(children=[
+            HTML(f"<b>Node Name:</b> {self.name}", style=style, layout=layout),
+            HTML(f"<b>Site:</b> {self.site}", style=style, layout=layout),
+            HTML(f"<b>Type:</b> {self.type}", style=style, layout=layout),
+            HTML(f"<b>Clock Speed:</b> {self.cpu['clock_speed'] / 1e9:.2f} GHz", style=style, layout=layout),
+            HTML(f"<b>RAM:</b> {self.main_memory['humanized_ram_size']}", style=style, layout=layout),
+            HTML(f"<b>GPU Model:</b> {self.gpu['gpu_model'] or 'None'}", style=style, layout=layout),
```

Comment:
This doesn't work for me when I use an openrc file from TACC, since not all nodes at TACC have `gpu_model` and `gpu_count` set.

Could you update the error handling to better support this? I think something like `self.gpu.get("gpu_count", 0)` would be good to default to 0. And if `gpu_count == 0`, don't include `gpu_model` in the display.